backends/aoti/runtime/AotiBackend.cpp

```diff
@@ -0,0 +1,561 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <executorch/extension/tensor/tensor.h>
+#include <executorch/runtime/backend/interface.h>
+#include <executorch/runtime/core/error.h>
+#include <executorch/runtime/core/evalue.h>
+
+#include <dlfcn.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include "cuda_runtime.h"
+
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+namespace executorch {
+namespace backends {
+namespace aoti {
+
+// Here is where the aoti bouncers are going to be defined.
+// I define the globals aoti generated compiled code calls
+// They can be backed by ET systems
+
+using namespace std;
+
+using executorch::aten::ScalarType;
```

Comment:
we shouldn't be hitting anything labeled aten because we're libtorch-free, right?